cmake/HPX_SetupCUDA.cmake

```diff
@@ -4,12 +4,15 @@
 # Distributed under the Boost Software License, Version 1.0. (See accompanying
 # file LICENSE_1_0.txt or copy at http://www.boost.org/LICENSE_1_0.txt)
 
-if((HPX_WITH_CUDA_COMPUTE OR HPX_WITH_ASYNC_CUDA) AND NOT TARGET Cuda::cuda)
+if(HPX_WITH_CUDA AND NOT TARGET Cuda::cuda)
```

Comment:
Are you sure about this? I think it's now possible to have `HPX_WITH_CUDA=OFF` but e.g. `HPX_:WITH_ASYNC_CUDA=ON`.

Reply:
Yes but with the change I made to `HPX_WITH_CUDA`, this option is now specifying just the language to use, so if `HPX_WITH_CUDA=OFF` but `HPX_WITH_ASYNC_CUDA=ON`, it means you are using HIP